### PR TITLE
sponsor detail page update, pathauto update, view and fields

### PR DESCRIPTION
**Pull request by vgrozdanoski**
Ready for styling

### CODE REVIEW

File: config/views.view.sponsors.yml [hunk 1]
```diff
@@ -704,14 +1158,84 @@ display:
...
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: target_id
+          type: image
+          settings:
+            image_style: ''
+            image_link: content
+            svg_render_as_image: false
```
[the remark this answers is not in the document]
Comment (vgrozdanoski) [reply config/views.view.sponsors.yml:1215]: I understood that we would like to present logos as svg and not in the  tag (src to svg), thus this is disabled on purpose. If we want to print the logos in  tag, that makes things easier in terms of resizing them from the backend. 

If that is the case, we need to provide dimensions for each sponsor level. For ex. Diamond `height:250px` (I would recommend that we limit the logos by height)

[the remark this answers is not in the document]
Comment (jdleonard) [reply config/views.view.sponsors.yml:1215]: I'd suggest img (src to svg). I suggest that we leave the heights up to @pbone . Presumably there will be different sizing on mobile.

**Comment by vgrozdanoski** [comment]
Maybe we can add this files in the .gitignore to prevent this kind of thing happening?

**Comment by jdleonard** [comment]
Those files should be in version control and thus not in .gitignore. Easier to discuss in person. Remind me next time and we can discuss.

**Comment by pbone** [comment]
subscribe, let me know when this gets merged into master

**Comment by jdleonard** [comment]
@pbone We need to keep master ready to deploy (e.g. for security updates) so this will not be merged until theming is complete. Please proceed in the associated feature branch. Let me know if I can help get you situated. Thanks!

**Comment by AmyJuneH** [comment]
https://www.drupal.org/project/drupalnyc/issues/3186092

**Pull request merged**